pyproject.toml

```diff
@@ -34,6 +34,16 @@ docs = [
     "sphinx_rtd_theme==1.2.2",
 ]
 
+[tool.black]
+line-length = 120
+target-version = ['py311']
+
+[tool.isort]
+atomic = true
+profile = "black"
+line_length = 120
+py_version=311
```

Comment:
So this [broke workflows](https://github.com/CSCfi/metadata-submitter/actions/runs/5642935238/job/15283779500?pr=781#step:7:18) and I have no idea why.

Reply:
it checks the test files from what i see:
so either we exclude them: https://github.com/CSCfi/metadata-submitter/actions/runs/5642935238/job/15283779500?pr=781#step:7:18 or some other fix